Tests/XCUITests/HistoryTests.swift

```diff
@@ -358,14 +352,13 @@ class HistoryTests: BaseTestCase {
     }*/
 
     // Smoketest
-    func testDeleteHistoryEntryBySwiping() {
+    func testDeleteHistoryEntryByContextMenu() {
```

Comment:
Swipe right no longer works, but we can remove the history by tapping "Delete from History" from the context menu.